test/lib/brick.js

```diff
@@ -1833,8 +1833,8 @@ if (typeof WeakMap === "undefined") {
     var LEFT_MOUSE_BTN = 0;
     var GET_DEFAULT_LABELS = function() {
         return {
-            prev: "←",
```

Comment:
Coulf you please revert this change, `brick.js` is a 3rd party lib.
As it is used in tests only, it should not affect you ?